### PR TITLE
fix(curriculum): combining steps 7-9 in build a football team cards

### LINKED ISSUE #55437: Combining steps 7-9 in build a football team cards
### Describe the Issue

steps 7-9 of build a football team cards project all ask for key-value pairs to be added to the project code.
They are simple and can easily be collected into one step with an instruction like the one below:

Below the team property, add three new property key-value pairs.
A key named `sport` and a string value of `"Football"`.
A key named `year` and a number value of `1986`.
A key named `isWorldCupWinner` and a boolean value set to `true`.

Note that step 6 also asks for a property to be added but we can leave that one as-is as a kind of 'warm-up' step (in case people forgot how to do it). While step 10 asks for another key-value pair but the text there is longer and so probably is best left as-is.

Alternatively, step 10 can be simplified if we rolled the initial instruction into the previous (newly combined) step.
So we could make the combined step say all the above plus this line:
A new key called `headCoach` with a value of an empty object.

Then in step 10 we can simplify the instruction down to:
Inside the headCoach object, add a property with a key of coachName and a string value of "Carlos Bilardo". Below that property, add another key called matches with a number value of 7.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-modern-javascript-methods-by-building-football-team-cards/step-7

### Your code

N/A

### Expected behavior

The newly combined step would have to provide test+hints for the three different properties being added.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55437

### CODE REVIEW

**Review by moT01 — CHANGES_REQUESTED** [review]
This removes two challenges, there should be changes in the meta.json file to go along with that I believe. We have some [helper scripts](https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#helper-scripts) that can maybe help with that.

**Comment by gagan-bhullar-tech** [comment]
@moT01  update meta.json file

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63cf93472de77d01bf8474bf.md [hunk 1]
```diff
@@ -309,6 +297,9 @@ const myFavoriteFootballTeam = {
   sport: "Football",
   year: 1986,
   isWorldCupWinner: true,
+  headCoach: {
```
Comment (moT01) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63cf93472de77d01bf8474bf.md:300]: The editable region should be moved to surround the `headCoach` object.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63cf90d4696d8f00851873a4.md [hunk 2]
```diff
@@ -23,6 +28,54 @@ Your `sport` property should be set to `Football`.
...
+assert.equal(myFavoriteFootballTeam.isWorldCupWinner, true);
+```
+
+Your `myFavoriteFootballTeam` object should have a `headCoach` property.
+
+```js
+assert.property(myFavoriteFootballTeam, 'headCoach');
+```
+
+Your `headCoach` value should be an object.
```
Comment (moT01) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63cf90d4696d8f00851873a4.md:73]: Kind of minor, but can we add a test or add to this test to make sure the object is empty?

**Review by moT01 — COMMENTED** [review]
This is looking good @gagan-bhullar-tech 👍 

I found a few more things we should fix up before getting this in.

**Comment by gagan-bhullar-tech** [comment]
@moT01 updated PR. Please review.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by Sembauke — APPROVED** [review]

**Pull request merged**